### PR TITLE
PIM-5922: fix scope switcher

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

Fix the scope dropdown position on the variant group screen

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Review and 2 GTM                  | N
| Micro Demo to the PO (Story only) | N

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Resources/public/js/datafilter/filter/product_scope-filter.js [hunk 1]
```diff
@@ -42,6 +42,10 @@ define(
              */
             moveFilter: function (collection) {
                 var $grid = $('#grid-' + collection.inputName);
+
+                if (0 === $grid.length) {
+                    $grid = $('[data-type="datagrid"]');
```
Comment (phaseinducer) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/datafilter/filter/product_scope-filter.js]: I think `$('[data-type="datagrid"]:first')` is safer here.

**Comment by phaseinducer** [comment]
GTM

**Comment by damien-carcel** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml [hunk 2]
```diff
@@ -73,6 +73,8 @@ error:
         attribute_option: Error during deletion of the attribute option
     saving:
         attribute_option: Cannot save attribute option
+        export_profile: Cannot save export profile. Please check that all filters are well filled
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: *correctly ?

**Pull request merged**